src/osp/Resource/load_tinygltf.cpp

```diff
@@ -350,3 +350,119 @@ ResId osp::load_tinygltf_file(std::string_view filepath, Resources &rResources,
     return res;
 }
 
+static phys::EShape shape_from_name(std::string_view name) noexcept
+{
+    using phys::EShape;
```

Comment:
This doesn't appear to do much for this function

Reply:
There's suppose to be more shapes added to this function; cube and cylinder are the only ones that exist in the gltf files so far.